src/components/Table/components/TableHeader/components/TableHeaderColumn/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import TableCell from '../../../TableCell';
+import { Props } from './types';
+
+
+/**
+ * The `<TableHeaderColumn />` component represents a single column in the `<TableHeader />`.
+ */
+function TableHeaderColumn({
+    children,
+    className,
+    label,
+    name,
+    ...otherProps
+}: Props): ReactElement {
+
+
+    const classes = clsx(
+        'brew-TableHeaderColumn',
+        { [`brew-TableHeaderColumn--${name}`]: Boolean(name) },
```

Comment:
I didn't see this anywhere in the SCSS file? Is it used for something else? Like being able to target a certain header? If so, can't ...otherProps be used for this?

Reply:
The thinking behind it is that it mirrors `brew-TableCell--column-${name}` in the body rows, allowing you to have predictable behavior and simplified DOM targeting for table sorting and such.

It may be overkill, so I'll bring it up for discussion tomorrow.

Reply:
I think I agree with @emdrayallday. The majority of use-cases probably won't leverage specific rows / cells, so we may not have the justification for all the class name bloat. In scenarios that they do need to target specific cells, they can use the more advanced prop passthrough method to place custom classes.

Reply:
And... done.